package.json

```diff
@@ -81,7 +81,9 @@
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-multi-entry": "^4.0.1",
     "@rollup/plugin-node-resolve": "^13.1.3",
+    "assetgraph-builder": "^9.0.0",
     "autoprefixer": "^9.8.6",
+    "canvas": "^2.11.2",
```

Comment:
Do we need to add `canvas` back too? Can we just revert the `assetgraph-builder` removal?

Reply:
`assetgraph-builder` includes the `canvas` as well, so it didn't make any difference in terms of annoyance

Reply:
Blurgh, 👍